openedx/adg/lms/applications/models.py

```diff
@@ -123,3 +137,18 @@ class Meta:
 
     def __str__(self):
         return 'WorkExperience {id}, for {organization}'.format(id=self.id, organization=self.name_of_organization)
+
+
+class AdminNote(TimeStampedModel):
+    """
+    Model to save the notes of admin on the user application.
+    """
+    user_application = models.ForeignKey(UserApplication, on_delete=models.CASCADE, )
+    admin = models.ForeignKey(User, on_delete=models.CASCADE, )
+    admin_note = models.TextField(verbose_name=_('Admin Note'), blank=True, )
```

Comment:
Can we please rename `admin_note` field to `note` as `admin_note` is redundant, keeping in mind the model name.

Reply:
Please update accordingly in the overriden `__str__` method

Reply:
Plus, in line 148, verbose_name is not needed because Django automatically creates it using the field's attribute name, converting underscores to spaces.

Reply:
Any particular reason behind `blank=True` in `admin_note` field? I don't think there is any point in storing an empty note in the database.